src/uikit.cs

```diff
@@ -12620,8 +12608,8 @@ interface UITableViewSource {
 		UITargetedPreview GetPreviewForDismissingContextMenu (UITableView tableView, UIContextMenuConfiguration configuration);
 
 		[NoWatch, NoTV, iOS (13,0)]
-		[Export ("tableView:willCommitMenuWithAnimator:")]
-		void WillCommitMenu (UITableView tableView, IUIContextMenuInteractionCommitAnimating animator);
+		[Export ("tableView:willPerformPreviewActionForMenuWithConfiguration:animator:")]
+		void WillPerformPreviewAction (UITableView tableView, UIContextMenuConfiguration configuration, IUIContextMenuInteractionCommitAnimating animator);
 
 		// WARNING: If you add more methods here, add them to UITableViewController as well.
```

Comment:
Is that warning needed? I know you didn't touch it but it's a scary warning

Reply:
Yup, that is needed so we do not forget to do that

Reply:
Buuuuuut you didn't do that then

Reply:
I did but you are right, the WARNING is wrong since what you want is `UITableViewControllerDelegate`

Reply:
```suggestion
		// WARNING: If you add more methods here, add them to UITableViewControllerDelegate as well.
```